Sources/SwifterSwift/UIKit/UIImageExtensions.swift

```diff
@@ -182,26 +182,40 @@ public extension UIImage {
         return newImage
     }
 
+    #if !os(watchOS)
     /// SwifterSwift: UIImage tinted with color
     ///
     /// - Parameters:
     ///   - color: color to tint image with.
     ///   - blendMode: how to blend the tint
     /// - Returns: UIImage tinted with given color.
     func tint(_ color: UIColor, blendMode: CGBlendMode) -> UIImage {
-        let drawRect = CGRect(x: 0.0, y: 0.0, width: size.width, height: size.height)
-        UIGraphicsBeginImageContextWithOptions(size, false, scale)
-        defer {
-            UIGraphicsEndImageContext()
+        let drawRect = CGRect(origin: .zero, size: size)
+        if #available(iOS 10.0, tvOS 10.0, *) {
+            let format = UIGraphicsImageRendererFormat()
+            format.scale = scale
+            let renderer = UIGraphicsImageRenderer(
+                size: size,
+                format: format
+            )
```

Comment:
This can all go on 1 line. In fact, if you look at the next function you can even avoid keeping this temporary reference.

Reply:
It makes sense to me 👍